addons/airway/functions/fnc_init.sqf

```diff
@@ -25,6 +25,8 @@ _unit setVariable [QGVAR(occluded), false, true];
 _unit setVariable [QGVAR(airway), false, true];
 _unit setVariable [QGVAR(overstretch), false, true];
 _unit setVariable [QGVAR(recovery), false, true];
+_unit setVariable [QGVAR(kingLTisSet), false, true];
```

Comment:
pls no, i already told you yesterday there is airway_item variable which can be used